feat: adds the defaults readme and updates other readmes (#39)
ZupIT/beagle-helpers: android/README.md

* If you wish to use Beagle for an application in production we advise you to configure a project
 from scratch using our

>  [documentation](https://docs.usebeagle.io/home/get-started/creating-a-project-from-scratch/case-android/)

Reviewer: ```suggestion
 [**documentation**](https://docs.usebeagle.io/home/get-started/creating-a-project-from-scratch/case-android/)
```

<hr>

### Getting started

> To start using Beagle right now, you can use the libs below:
> * [Beagle-Scaffold](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-scaffold):
> This lib will hold almost all configurations needed to start using Beagle in your project.

Reviewer: ```suggestion
To start using Beagle right now, you can use the libs below:
* [**Beagle-Scaffold**](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-scaffold)
This lib will take care of almost all configurations needed to start using Beagle in your project.
```

> We advise the use of these libs for people that never used Beagle before.

Reviewer: ```suggestion
We recommend the use of this lib for people that are just starting to learn Beagle.
```

> * [Beagle-Defaults](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-defaults):

Reviewer: ```suggestion
* [**Beagle-Defaults**](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-defaults):
```

> This lib is advised for more experienced Beagle users and 
> will hold only a few class configurations necessary to 
> use Beagle in an application. These configurations include 
> default HttpClient, Logger and Cache classes.<br>
> All these are also available in the Scaffold lib above.

Reviewer: ```suggestion
This lib is advised for more experienced Beagle users as it only takes care of a few class configurations necessary to use Beagle in an application. These configurations include a default HttpClient, Logger and Cache classes.<br> 
All these are also available in the Scaffold lib above.
```

* If you wish to use Beagle for an application in production we advise you to configure a project
 from scratch using our
 [documentation](https://docs.usebeagle.io/home/get-started/creating-a-project-from-scratch/case-android/)

<hr>

### Getting started

To start using Beagle right now, you can use the libs below:
* [Beagle-Scaffold](https://github.com/ZupIT/beagle-helpers/tree/main/android/beagle-scaffold):
This lib will hold almost all configurations needed to start using Beagle in your project.

> We advise to use these libs for people that never used Beagle before.

Reviewer 2: ```suggestion
We advise the use of these libs to people that never used Beagle before.
```
